controls/roles/validator-import/molecule/default/verify.yml

```diff
@@ -0,0 +1,16 @@
+---
+- name: Verify
+  hosts: all
+  gather_facts: false
+  tasks:
+  - stat: path=/etc/stereum/services
+    register: stereum_services_dir
+  - debug:
+      msg: "{{ stereum_services_dir }}"
+  - shell: docker ps
+    register: stereum_docker_ps
+  - debug:
+      msg: "{{ stereum_docker_ps }}"
+  - assert:
+      that:
+      - stereum_docker_ps.stdout.find("stereum/lighthouse") != -1
```

Comment:
can we check somehow that the validator is loading the keys correctly? or at least that it connected to the beacon node?

Reply:
done!